### PR TITLE
Chicago, Paris, and Washington DC launch (including Premium plan details)

### PR DESCRIPTION
**Pull request by wildmanonline**
This is the PR for the documentation to support the new data center releases. This includes the launch of premium plans and details the upgraded OBJ capacity in these new data centers.

### CODE REVIEW

File: docs/products/compute/compute-instances/guides/lish/index.md [hunk 1]
```diff
@@ -175,31 +174,21 @@ ssh -t [manager-username]@lish-[location].linode.com [linode-name]
...
-| Frankfurt (Germany) | `lish-frankfurt.linode.com` | `frankfurt.webconsole.linode.com` |
-| Fremont (California, USA) | `lish-fremont.linode.com` | `fremont.webconsole.linode.com` |
-| London (United Kingdom) | `lish-london.linode.com` | `london.webconsole.linode.com` |
-| Mumbai (India) | `lish-mum1.linode.com` | `mum1.webconsole.linode.com` |
-| Newark (New Jersey, USA) |  `lish-newark.linode.com` | `newark.webconsole.linode.com` |
-| Singapore | `lish-singapore.linode.com` | `singapore.webconsole.linode.com` |
-| Sydney (Australia) | `lish-syd1.linode.com` | `syd1.webconsole.linode.com` |
-| Tokyo (Japan) | `lish-tokyo2.linode.com` <br>or `lish-shg1.linode.com` | `shg1.webconsole.linode.com` |
-| Toronto (Canada) | `lish-tor1.linode.com` | `tor1.webconsole.linode.com` |
+Each data center has its own gateways, which provides access to Lish, Weblish, and Glish. When connecting through Lish, To connect to one of these services, use the corresponding gateway in any data center, but we recommend using one close to the data center it’s located within. The gateway boxes are available over IPv4 and IPv6.
```
Comment (andystevensname) [inline docs/products/compute/compute-instances/guides/lish/index.md:191]: lowercase `to`, but one of those sentence intros could be removed. Either `When connecting through Lish,` or  `To connect to one of these services,`

Also, I think `but` can be removed and what follows turned into its own sentence.

File: docs/products/compute/compute-instances/guides/lish/index.md [hunk 2]
```diff
@@ -209,127 +198,223 @@ If you are having issues accessing Lish, Weblish, or Glish, you may be blocked b
...
+    RSA 59:30:1a:0b:93:5e:3f:4b:6f:d1:96:ff:7e:9e:12:f8
+    ECDSA SHA256:8emv5PuUgPB2GFejMYWl1f4x1yj3YqAQPAYIrBm43ZI
+    Ed25519 SHA256:7k2c442k+zqbGaraZvmqXM3MA5lCcthaR2lbrB651lg
+    Ed25519 MD5:b3:76:56:3a:d8:66:5e:0c:61:90:03:b7:0b:ca:58:25
+    ```
+    {{< /note >}}
 
-These are the fingerprints for the Lish gateway in our Atlanta data center (lish-atlanta.linode.com):
+-   **Weblish Gateway:** `atlanta.webconsole.linode.com`
+-   **Glish Gateway:**
```
Comment (andystevensname) [inline docs/products/compute/compute-instances/guides/lish/index.md:215]: Missing gateway

File: docs/products/compute/compute-instances/guides/lish/index.md [hunk 3]
```diff
@@ -209,127 +198,223 @@ If you are having issues accessing Lish, Weblish, or Glish, you may be blocked b
...
+    RSA 6d:3d:b5:d0:42:1c:49:45:a6:47:29:bd:88:4e:58:d4
+    ECDSA SHA256:1fL1HTGas1APIpshCu1ZWys7LI97s8eTFN3+f8zEYXA
+    Ed25519 SHA256:Ime9h7IAxAjBvMGR+G+EnbaLWpvXO+Z7TaGHzM9g5Sc
+    Ed25519 MD5:23:c1:02:ba:40:77:f7:73:0b:d4:0f:3a:f1:8c:74:b6
+    ```
+    {{< /note >}}
 
-#### Fremont
+-   **Weblish Gateway:** `dallas.webconsole.linode.com`
+-   **Glish Gateway:**
```
Comment (andystevensname) [inline docs/products/compute/compute-instances/guides/lish/index.md:246]: missing glish gateway for a number of DCs

**Review by andystevensname — APPROVED** [review]

File: docs/products/compute/compute-instances/guides/lish/index.md (hunk 1 above)
Comment (wildmanonline) [reply docs/products/compute/compute-instances/guides/lish/index.md:191]: I reworked this paragraph to fix these issues.

File: docs/products/compute/compute-instances/guides/lish/index.md (hunk 3 above)
Comment (wildmanonline) [reply docs/products/compute/compute-instances/guides/lish/index.md:246]: I added these back in. Thanks!

**Pull request merged**